src/styles/blocks/card.css

```diff
@@ -0,0 +1,13 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
```

Comment:
Set font-family for the whole page with html selector, don't repeat yourself in every class